lib/beaker/shared/host_manager.rb

```diff
@@ -1,3 +1,5 @@
+require 'in_parallel'
```

Comment:
it looks like this require is set here to be a one-off, or to limit the scope of this change. But we're changing `run_block_on`, so there's no real limit to the scope of this change :). Is there another reason that this require is here rather than with the others?